Formula/maestro@head.rb

```diff
@@ -1,10 +1,10 @@
 class MaestroHead < Formula
-  desc "Maestro CLI"
+  desc "The simplest and most effective mobile UI testing framework."
   homepage "https://maestro.mobile.dev"
   license "Apache-2.0"
 
   conflicts_with "mobile-dev-inc/tap/maestro", because: "This is the head version of maestro"
-  depends_on "openjdk@11"
+  depends_on "openjdk"
```

Comment:
Was this intentional?

Reply:
Yes, there's no need to enforce older JDK. 

It's a convention across common Formulas on homebrew - if they depend on Java, they do it like this.